.github/workflows/build-and-push-images.yaml

```diff
@@ -89,6 +89,9 @@ jobs:
             platforms: linux/amd64,linux/arm64
 
     steps:
+      - name: Checkout
```

Comment:
@mahdikhashan Do you want to address this ask in the next PR?
https://github.com/kubeflow/trainer/pull/2455#discussion_r1997645083

Reply:
I'll create an issue for it, if its fine with you.

Reply:
Sure, thanks!

Reply:
https://github.com/kubeflow/trainer/issues/2532